Tests/SentryTests/UIRedactBuilderTests.swift

```diff
@@ -0,0 +1,130 @@
+#if os(iOS)
+import Foundation
+import Nimble
+@testable import Sentry
+import UIKit
+import XCTest
+
+class UIRedactBuilderTests: XCTestCase {
```

Comment:
Those tests are the specification to the sut. I don't need to read the code of the sut to understand what the class does. Excellent 🥇